rules/windows/registry_event/sysmon_modify_screensaver_binary_path.yml

```diff
@@ -0,0 +1,29 @@
+title: Path To Screensaver Binary Modified
+id: 67a6c006-3fbe-46a7-9074-2ba3b82c3000
+status: experimental
+description: >
+  Detects value modification of registry key containing path to binary used as screensaver. 
+  Adversaries may establish persistence by executing malicious content triggered by user inactivity. 
+  Screensavers are programs that execute after a configurable time of user inactivity and consist of Portable Executable (PE) files with a .scr file extension.
```

Comment:
Hello @bczyz1 !
A good rule! It is surprising that such a rule has not yet been in SIGMA. :)
I recommend:
shorten the description to one sentence (it contains the detection logic)

Reply:
Hey @e6e6e, thanks, followed your recommendations, see 8ae42bca7ceb3f06bc73b117f1ea48e11f06d803.